### PR TITLE
Void cells actually produce power now.

### PR DESCRIPTION
**Pull request by Wilchenx**
:cl: Void cells actually recharge now.
add: Void cell self_recharge
/:cl:

The description is correct now.

### CODE REVIEW

File: code/modules/power/cell.dm [hunk 1]
```diff
@@ -280,6 +280,7 @@
 	maxcharge = 50000
 	rating = 12
 	ratingdesc = FALSE
+	self_recharge = 1
```
Comment (RemieRichards) [inline code/modules/power/cell.dm]: but they're ```/infinite```, which means their ```use``` just returns ```1```/```TRUE```, they never go down in power.

Comment (The-clown-known-as-Pepe) [inline code/modules/power/cell.dm]: Unless you emp them, same with infinite power cells.

Comment (RemieRichards) [inline code/modules/power/cell.dm]: That only halves maxcharge, they still won't use power.
An infinite cell with no power will still work anyway, as again, it's ```use()``` proc doesn't even check if it has the charge necessary to work, it just -works-

Comment (SgtxRootbeer) [inline code/modules/power/cell.dm]: doesn't the SMES check the stored power inside the cell? or am i thinking of this wrong?

Comment (RemieRichards) [inline code/modules/power/cell.dm]: Then that is the SMES's fault, it should just check if it can ```use(1)``` or something.

Comment (GunHog) [inline code/modules/power/cell.dm]: Why not just make it emp-proof?

Comment (RemieRichards) [inline code/modules/power/cell.dm]: @GunHog As I said, EMPs don't actually do anything to infinite cells.
The SMES issue wouldn't be solved by preventing EMPs as the infinite cell will be treated as having 50,000 charge, not inifnite, which is still wrong.

Comment (The-clown-known-as-Pepe) [inline code/modules/power/cell.dm]: Well it still corrects the code to fit the description, which should correct any other silly interactions with emp.

Comment (RemieRichards) [inline code/modules/power/cell.dm]: It corrects nothing.

**Comment by Repukan** [comment]
Void cells?

**Comment by Kromgar** [comment]
@Repukan I beleive they are alien power cells

**Comment by The-clown-known-as-Pepe** [comment]
Correct, they are called abductor in code but are "Void cells" ingame.

**Comment by Kromgar** [comment]
@The-clown-known-as-Pepe is there any way to get them ingame?

**Comment by The-clown-known-as-Pepe** [comment]
@Kromgar Lots of luck and experimentor. :^)

**Comment by Kromgar** [comment]
@The-clown-known-as-Pepe So if you research a void cell can you print them?

**Comment by The-clown-known-as-Pepe** [comment]
Nope, experimentor only.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Something needs to be done about @RemieRichards comments rather that building on a broken system

**Pull request closed without merging**